### ISSUE TITLE
META: Timeline visualization improvements

### ISSUE THREAD

**Issue by roamye** [project-lux/lux-frontend#487]
**Problem Description:**
This ticket will serve as the meta ticket for all improvements related to the new feature, `timeline visualizations`. 

**Expected Behavior/Solution:**
All issues below will be completed to improve the timeline visualizations. 

**Requirements:**
Complete all children issues to improve the timeline visualizations 

**Child Github Issues:**

- Issues that contain similar work but are not blocking or being blocked by the current issue.

**Related links:**
- [POW recommendations](https://www.figma.com/proto/bWS2orxFECNrG92HfG48BL/LUX-Yale-Design-Walkthrough?node-id=3916-4088&node-type=frame&t=yGmZEMr8Qr7RkiBS-0&scaling=min-zoom&content-scaling=fixed&page-id=3898%3A7)

**Comment by roamye** [comment]
From accessibility meeting (notes): 
- Adding a pattern or texture to the timeline visualizations is recommended to address color contrast issues.
- Recharts make this difficult to implement, but Nick will investigate. 
- New ticket will be made after his investigation.
